### ISSUE TITLE
Feature Proposal: Neural Extensions

### ISSUE THREAD

**Issue by FeepingCreature** [bo0ts/ddate#18]
Just for fun, and since I got a bit bored of the five weekdays, I've created a set of 72 additional weekdays by feeding the five days into https://talktotransformer.com as a list. A version with this list is at https://github.com/FeepingCreature/ddate/commit/7421de439df564bfbf212cb42bca57afd5dbe393 ; they start neat but go increasingly weird as the network loses track of what sort of thing the list is for.

Right now this is a hardcoded, hand-generated list, I'm think of regenerating them every season and sticking the file on a public webserver somewhere so there's a new random set of extended day names every time. If I don't get around to that though, I still wanted to bring the idea up. :)

**Comment by danwdart** [comment]
That's amazing. Well, why don't we have a day name for every distinct day, generated in the same way?

**Comment by FeepingCreature** [comment]
That's effectively what it comes down to.

I generally reroll the days on my fork every season. But some of the previous seasons have been very silly to very boring; check the commit log at https://github.com/FeepingCreature/ddate/commits/master for examples. I think the current one is ... *usable*, even though it got into a bit of a rut.

**Comment by emberdart** [comment]
I feel like this is a fork-feature and probably an ifdef, since it's not in the original spirit of how these things were invented and supposed, but the antithesis to this is X-Day.

If i were to outright say no, "they" would absolutely show me the colour of my face.

Let's remove or replace a couple of those, because they are definitely going to get some of us in tremendous trouble anyway.

**Comment by FeepingCreature** [comment]
Yeah I've been updating it every discordian season since then on my fork, taking whatever whichever AI service I tried that month gave me. There's definitely been some fun ones and some very dubious ones, but it's been tremendous fun personally.

Probably not something that can be integrated upstream all things considered for many reasons, like that it's fundamentally down to taste- way too many variables, like what network you use and at what settings. And yes, if you get the network into basemodel mode, it has no political restraints on day names. I can strongly recommend doing it yourself if you have the time, at least.

Considered as an upstream feature, it should probably read the list of extended days from /usr/shared or .local, and leave the job of updating it to the user. To me the whole point is that it's not a fixed list, it's like a fortune cookie every day.
